### PR TITLE
[14.0][FIX] stock_orderpoint_origin: support multiwarehouse case

### PR DESCRIPTION
**Pull request by dreispt**
.

### CODE REVIEW

**Review by patrickrwilson — APPROVED** [review]
LGTM

File: stock_orderpoint_origin/models/procurement_group.py [hunk 1]
```diff
@@ -15,9 +15,10 @@ def run(self, procurements, raise_user_error=True):
         Forecast = self.env["report.stock.report_product_product_replenishment"]
         new_procurements = []
         for procurement in procurements:
+            warehouse = procurement.values["warehouse_id"]
+            ForecastWH = Forecast.with_context(warehouse=warehouse.id)
```
Comment (rousseldenis) [inline stock_orderpoint_origin/models/procurement_group.py]: I think warehouse can be void too. That has to be taken into account.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
@dreispt @patrickrwilson I think it's time to add tests on this in order to avoid regressions. Thanks

File: stock_orderpoint_origin/models/procurement_group.py (hunk 1 above)
Comment (dreispt) [reply stock_orderpoint_origin/models/procurement_group.py]: @rousseldenis Done

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**